CHANGELOG.md

```diff
@@ -92,6 +92,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * `jj git import` no longer crashes when all Git refs are removed.
 
+* Git submodules are now ignored completely. Earlier the files in a submodule
+  are either added to the working copy or deleted when checking out a tree.
```

Comment:
I think we should clarify that "the files in a submodule" are "the files in the submodule in the working copy". Maybe rewrite it a bit, like this:

Git submodules are now ignored completely. Earlier, files present in the submodule directory in the working copy would become added (tracked), and later removed if you checked out another commit. You can now use `git` to populate the submodule directory and `jj` will leave it alone.